test/SAPHanaSR-upgrade-to-angi-demo

```diff
@@ -55,7 +55,7 @@ clone @@clntop@@ rsc_SAPHanaTop_@@sid@@_HDB@@ino@@ \
 CIB_CLNTMP_FIL="#
 primitive rsc_SAPHanaFil_@@sid@@_HDB@@ino@@ ocf:suse:SAPHanaFilesystem \
   op start interval=0 timeout=10 \
-  op stop interval=0 timeout=20 on-fail=fence \
```

Comment:
It depends on the use-case and on the expectation. If even SAP HANA for ScaleUp is installed on NFS, it might still make sense.
In our man page we have it with (ScaleOut) and without.

Reply:
Hi Fabian,

the trick of SAPHanaFilesystem is its overall parameter ACTION_ON_FAIL=fence, which is the default. That means, the stop operation will be bypassed in case od srHook=SOK.

The OCF stop operation´s option on-fail=fence is default. However, it would not consider srHook at all. This option is not needed for SAPHanaFilesystem.

The on-fail=fence is just a copy-and-paste typo from the very beginning. I do not know how it managed to come back into the script.

Regards,
Lars